Mathlib/Data/ENNReal/Basic.lean

```diff
@@ -223,6 +223,12 @@ theorem ofReal_eq_coe_nnreal {x : ℝ} (h : 0 ≤ x) :
 
 @[simp] theorem toReal_nonneg {a : ℝ≥0∞} : 0 ≤ a.toReal := a.toNNReal.2
 
+-- Note: Tagging this with `@[simp]` causes loops, but why?
```

Comment:
Probably you want to tag this `@[norm_cast]`, not `@[simp]`.

Reply:
Thanks! I was not (actively) aware of `@[norm_cast]` tag, but it definitely seems appropriate. I will try and see the consequences (especially no loops).